content/applications/marketing/events/event_tracks.rst

```diff
@@ -0,0 +1,231 @@
+
+Then, ensure the track is applied to the correct event in the :guilabel:`Event` field. By default,
+this field is auto-populated with the event related to the *Event Tracks* page the track was
+originally created from.
+
+Next, choose to add existing tags, or create new ones, to further organize the track. These tags,
+and their corresponding tag categories are utilized on the event specific website - mainly on the
+*Talks* web page on the event website, via the drop-down filter menus.
+
+Beneath that general information section, there are three tabs: :guilabel:`Speaker`,
+:guilabel:`Description`, and :guilabel:`Interactivity`.
+
+Speaker tab
+~~~~~~~~~~~
+
+The :guilabel:`Speaker` tab on an event track form is filled with various fields related to the
+specific speaker who is planning to conduct/host the track.
+
+.. image:: event_tracks/speaker-tab.png
+   :align: center
+   :alt: The Speaker tab on an event track form in the Odoo Events application.
+
+Contact Details section
+***********************
+
+In the :guilabel:`Contact Details` section, proceed to use the :guilabel:`Contact` drop-down field
+to select an existing contact from the database as the main point of contact for the talk.
+
+If this contact is not yet in the database, type in the name of the contact, and click
+:guilabel:`Create` to create and edit the contact form later, or click :guilabel:`Create and
+edit...` to be taken to that new contact's contact form, where the rest of their pertinent
+information can be entered.
+
+The :guilabel:`Contact Email` and :guilabel:`Contact Phone` fields are greyed-out and populated with
+the information found on that chosen contact's contact form. These fields are not modifiable once
+the :guilabel:`Contact` field is selected.
+
+Speaker Bio section
+*******************
+
+In the :guilabel:`Speaker Bio` section, proceed to enter information related to the specific speaker
+scheduled to conduct/host the track. This section may auto-populate based on the :guilabel:`Contact`
+selected in the :guilabel:`Contact Details` section. If not, enter information in the necessary
+fields.
+
+.. note::
+   This information appears on the front-end of the event website, on the specific track webpage,
+   providing more information about the speaker to the track attendees.
+
+Start by entering a :guilabel:`Name`, :guilabel:`Email`, and :guilabel:`Phone` number for the speaker.
+
+Next, if desired, add an image to appear alongside the speaker biogrpahy on the event website, via
+the :icon:`fa-pencil` :guilabel:`(pencil)` icon that appears when the cursor hovers over the
+:icon:`fa-camera` :guilabel:`(camera)` icon. When clicked, proceed to upload the desired image for
+the speaker.
+
+Then, enter a :guilabel:`Job Position` for the designated speaker, followed by the
+:guilabel:`Company Name` associated with the speaker.
+
+In the :guilabel:`Biography` field, proceed to enter in a custom biography with any speaker-related
+information.
```

Comment:
Is there anything else they may need to know- for example, can they write in HTML and it populates the website like that? or anything they can or can't do? I'm just thinking out loud!

Reply:
I completely get where you're coming from with this suggestion, however, I think keeping things simple and succinct is the best move here, as it's a fairly self-explanatory field. But, again, I really appreciate the suggestion. 🙏